src/plugins/plot/barGraph/BarGraphPlot.vue

```diff
@@ -220,9 +248,51 @@ export default {
             this.isZoomed = false;
             this.$emit(SUBSCRIBE);
         },
+        barColorChanged() {
+            const plotUpdate = {
+                marker: {
+                    color: this.domainObject.configuration.barStyles.color
+                }
+            };
+            Plotly.restyle(this.$refs.plot, plotUpdate);
+        },
+        fontChanged() {
+            const font = this.processFontConfig();
+
+            const plotUpdate = {
+                font
+            };
+
+            Plotly.restyle(this.$refs.plot, plotUpdate);
+        },
         updateData() {
             this.updatePlot();
         },
+        processFontConfig() {
+            const font = {
+                family: DEFAULT_FONT_FAMILY
+            };
+
+            const objectStyles = this.domainObject.configuration.objectStyles;
+            if (objectStyles && objectStyles.staticStyle.style && objectStyles.staticStyle.style.color) {
```

Comment:
I believe font colors can also be specified in objectStyles. Did you not have luck with that?
Also, there is a objectStyles.conditionalStyle property as well that you'll need to handle here.

Reply:
@shefalijoshi is taking a look